services-camel/src/main/resources/config.properties

```diff
@@ -9,6 +9,7 @@ fcrepo.authPassword=
 fcrepo.authHost=
 
 cdr.enhancement.thumbnail=PNG
+cdr.enhancement.fulltext=_fullText.txt
```

Comment:
can you rename this property here and in its usage to "cdr.enhancement.fulltext.fileName", and also since its not a prefix anymore it should be formatted like the value you use in the test.  Should also rename `cdr.enhancement.thumbnail` to something that indicates what property it is.